website/blog/2023-12-23-AgentOptimizer/index.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: AgentOptimizer - An Agentic Way to Train Your LLM Agent
+authors:
+  - skzhang1
+  - jieyuz2
+tags: [LLM, research]
+---
+
+![Overall structure of AgentOptimizer](img/agentoptimizer.png)
+
+
+**TL;DR:**
+Introducing **AgentOptimizer**, a new class for training LLM agents in the era of LLMs as a service.
+**AgentOptimizer** is able to prompt autogen agents to iteratively optimize its function/skills according to the historical conversation and performance.
+Checkout one implementation for **AgentOptimizer** on [MATH](https://github.com/hendrycks/math) dataset
```

Comment:
This sentence seems to indicate that `AgentOptimizer` is dataset-dependent. Is this true? Please clarify.

Reply:
Need to be more clear about how general this class is.

Reply:
Yes, it is dataset-dependent for now. Currently it is mainly for illustration purpose and a proof of concept. I clarified it in the end.

Reply:
resolved